### ISSUE TITLE
[Test bug] DISABLED_l2_arp_test_one_machine not passing

### ISSUE THREAD

**Issue by er1cthe0ne** [futurewei-cloud/alcor-control-agent#223]
The DISABLED_l2_arp_test_one_machine is not passing on our lab machine setup, the command used below (machine IP is 10.213.43.187):

./build/tests/aca_tests --gtest_also_run_disabled_tests --gtest_filter=*DISABLED_l2_arp_test_one_machine -c 10.213.43.187

Below is the relevent logs, the failure could be caused by the fact that we didn't pass in port configuration into the goal state? We can consider removing this test case if the scenario is already covered by other tests.

Starting to update goal state with format_version: 0
[METRICS] Goal state message size is: 304 bytes
parsed_struct.subnet_states(0).operation_type(): INFO
current_SubnetConfiguration.revision_number(): 1
current_SubnetConfiguration.request_id(): 
current_SubnetConfiguration.id(): 27330ae4-b718-11ea-b3de-111111111111
current_SubnetConfiguration.update_type(): 0
current_SubnetConfiguration.network_type(): 0
current_SubnetConfiguration.vpc_id(): 1b08a5bc-b718-11ea-b3de-111111111111
current_SubnetConfiguration.name():  
current_SubnetConfiguration.cidr(): 10.10.0.0/24 
current_SubnetConfiguration.tunnel_id(): 123 
current_SubnetConfiguration.gateway().ip_address(): 10.10.0.1 
current_SubnetConfiguration.gateway().mac_address(): fa:16:3e:d7:f2:11 
current_SubnetConfiguration.dhcp_enable(): 0 
current_SubnetConfiguration.availability_zone():  

parsed_struct.neighbor_states(0).operation_type(): CREATE
current_NeighborConfiguration.revision_number(): 1
current_NeighborConfiguration.id(): 03333333-b718-11ea-b3de-111111111111
current_NeighborConfiguration.request_id(): 
current_NeighborConfiguration.update_type(): 0
current_NeighborConfiguration.vpc_id(): 1b08a5bc-b718-11ea-b3de-111111111111
current_NeighborConfiguration.name():  
current_NeighborConfiguration.mac_address(): fa:16:3e:d7:f2:6e 
current_NeighborConfiguration.host_ip_address(): 172.16.62.158 
current_NeighborConfiguration.fixed_ips_size(): 1 
current_NeighborConfiguration.fixed_ips(0): neighbor_type: 0, subnet_id 27330ae4-b718-11ea-b3de-111111111111, ip_address 10.10.0.103 

... 

Receiving arp message from inport=1
ARP entry exist (ip = 10.10.0.103 and vlan id = 1) with mac = fa:16:3e:d7:f2:6e
PING 10.10.0.103 (10.10.0.103): 56 data bytes
64 bytes from 10.10.0.103: seq=0 ttl=64 time=0.718 ms

--- 10.10.0.103 ping statistics ---
1 packets transmitted, 1 packets received, 0% packet loss
round-trip min/avg/max = 0.718/0.718/0.718 ms
size_udp < 20: 0 bytes
Command succeeded!
 Elapsed time for system command took: 177266 microseconds or 177 milliseconds.
Executing command: docker exec con2 ping -c1 10.10.0.101
PING 10.10.0.101 (10.10.0.101): 56 data bytes

--- 10.10.0.101 ping statistics ---
1 packets transmitted, 0 packets received, 100% packet loss
Command failed!!! rc: 256
 Elapsed time for system command took: 10169982 microseconds or 10169 milliseconds.
/root/aca-dev/test/gtest/aca_test_arp.cpp:303: Failure
Expected equality of these values:
  overall_rc
    Which is: 256
  0

**Comment by lly00** [comment]
Since the code `set_host_ip_address("172.16.62.158")` set the host ip as my computer's ip, the test do not actually update L2 neighbor info. However, it will have an adverse effect on other test nodes. One way to solve this problem is to remove all the configuration steps since they are not necessary.(or just remove the test case)

**Comment by lly00** [comment]
Since the code `set_host_ip_address("172.16.62.158")` set the host ip as my computer's ip, the test do not actually update L2 neighbor info. However, it will have an adverse effect on other test nodes. One way to solve this problem is to remove all the configuration steps since they are not necessary.(or just remove the test case)

**Comment by er1cthe0ne** [comment]
> Since the code `set_host_ip_address("172.16.62.158")` set the host ip as my computer's ip, the test do not actually update L2 neighbor info. However, it will have an adverse effect on other test nodes. One way to solve this problem is to remove all the configuration steps since they are not necessary.(or just remove the test case)

Yes, I believe this ARP test case is already covered by other ARP and traffic test cases. If you agree, then please remove this test case. Thanks.
